lamindb_setup/core/_settings_instance.py

```diff
@@ -465,6 +465,13 @@ def _init_db(self):
         setup_django(self, init=True)
         _check_setup.IS_LOADING = False
 
+        from lamindb.models import Space
```

Comment:
I suggest to move it into init, because persists is also called in connect. I think this is problematic in connect because it slows down and also can error if lamindb is not reloaded in connect (cli call).

Reply:
Ok, sorry, it is not in persists, please ignore.

Reply:
It is in `_init_db`. That's only called during init (`init=True`). Isn't it?

Reply:
Yes, correct, my mistake, I thought it was in persists not in init_db.

Reply:
The problem is that the other to records that are created in lamindb-setup (User & Storage) are touched on connect. This here is only touched on init.

Sorry, I didn't see this before!